app/Policies/TopicPolicy.php

```diff
@@ -15,7 +15,7 @@ class TopicPolicy
 
     public function viewAny(MoonshineUser $user): bool
     {
-        if ($user->moonshineUserRole->id === Constants::ROLES['Admin']) {
+        if ($user->moonshineUserRole->id !== Constants::ROLES['Developer']) {
```

Comment:
It was approved that developers don't have access to topics at all, please refer to your roles.csv file.